<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tablo</class>
 <widget class="QDialog" name="Tablo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tablo_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1041</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tutar_lbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>740</y>
     <width>981</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>BEKLENİYOR ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>40</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Sil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ek_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>110</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pdf_btn">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>180</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PDF Görüntüle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pdf_ekle_btn">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>250</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PDF Ekle</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
